### PR TITLE
Some leaked M3C cards

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/final act.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Final Act
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/final act.txt:1]: The filename is missing a _

File: forge-gui/res/cardsfolder/upcoming/broodmate_tyrant.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Broodmate Tyrant
+ManaCost:4 B R G
+Types:Creature Dragon
+PT:5/5
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a 5/5 red Dragon creature token with flying.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$  r_5_5_dragon_flying | TokenOwner$ You
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/broodmate_tyrant.txt:7]: spacing

**Comment by paulsnoops** [comment]
@Fulgur14 Are you checking the projects? Some of these were already claimed but it's fine we can leave them in but please make sure to check going forward to save duplicate work :)

File: forge-gui/res/cardsfolder/upcoming/eldrazi_confluence.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Eldrazi Confluence
+ManaCost:2 C C
+Types:Sorcery
+A:SP$ Charm | Choices$ DBSpatialContortion,DBEldraziDisplacer,DBEldraziScion | CharmNum$ 3 | CanRepeatModes$ True
+SVar:DBSpatialContortion:DB$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ +3 | NumDef$ -3 | IsCurse$ True | SpellDescription$ Target creature gets +3/-3 until end of turn.
+SVar:DBEldraziDisplacer:DB$ ChangeZone | TargetMin$ 1 | ValidTgts$ Permanent.nonLand | TgtPrompt$ Select target nonland permanent | RememberTargets$ True | Origin$ Battlefield | Destination$ Exile | SubAbility$ DBReturn | SpellDescription$ Exile target nonland permanent, then return it to the battlefield under its owner's control.
+SVar:DBReturn:DB$ ChangeZone | Defined$ Remembered | Origin$ Exile | Tapped$ True | Destination$ Battlefield
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/eldrazi_confluence.txt:7]: cleanup missing

**Comment by tool4ever** [comment]
@paulsnoops
should we wait until the leaks get confirmed? 🤔

**Comment by paulsnoops** [comment]
> @paulsnoops should we wait until the leaks get confirmed? 🤔

Nah we can merge them if you want? I think they're legit and we can always delete them if needed imo

**Pull request closed without merging**